src/main/java/com/powsybl/openloadflow/network/impl/LfNetworks.java

```diff
@@ -73,11 +74,16 @@ public void visitThreeWindingsTransformer(ThreeWindingsTransformer transformer,
 
                 @Override
                 public void visitGenerator(Generator generator) {
-                    if (!Objects.equals(generator.getRegulatingTerminal().getBusView().getBus().getId(),
-                            generator.getTerminal().getBusView().getBus().getId())) {
-                        throw new PowsyblException("Generator remote voltage control is not yet supported");
-                    }
                     lfBus.addGenerator(generator);
+
+                    String controlBusId = generator.getRegulatingTerminal().getBusView().getBus().getId();
+                    String connectedBusId = generator.getTerminal().getBusView().getBus().getId();
+                    if (!Objects.equals(controlBusId, connectedBusId)) {
```

Comment:
Are we sure that we want to do that even if the voltage regulation is disabled ? In my use case, if I remove the remote voltage control from generators, there is no calculation.